### PR TITLE
RK-12854 - Try to fix language server issues

### PR DESCRIPTION
**Pull request by Urook**
Try to fix find go location. Try to fix python language server for windows.

### CODE REVIEW

File: src/langauge-servers/pythonUtils.ts [hunk 1]
```diff
@@ -1,17 +1,14 @@
...
 import { getLogger } from "../logger";
 
 const logger = getLogger("langserver");
-export const isWindows = process.platform.match("win32");
+const isWindows = process.platform.match("win32");
 const isMac = process.platform.match("darwin");
 const isLinux = process.platform.match("linux");
-export const PYTHON_FILENAME = isWindows ? "python3.exe" : "python3";
-export const PIP_FILENAME = isWindows ? "pip3.exe" : "pip3";
+export const PYTHON_FILENAME = isWindows ? "python.exe" : "python3";
```
Comment (DavidRBiderman) [inline src/langauge-servers/pythonUtils.ts:11]: change to PYTHON_EXEC

File: src/langauge-servers/configStore.ts [hunk 2]
```diff
@@ -190,16 +186,26 @@ class LangServerConfigStore {
...
-            const stdout = cp.execFileSync(PIP_FILENAME, ["show", "python-lsp-server"], { cwd: this.serverLocations["python"], encoding: "utf-8" });
+            const stdout = cp.execFileSync(
+                PYTHON_FILENAME,
+                ["-m", "pip", "show", "python-lsp-server"],
+                { cwd: this.serverLocations["python"], encoding: "utf-8" }
+            );
             const trimmedOutput = _.trim(stdout);
             if (trimmedOutput.startsWith("WARNING: Package(s) not found:")) {
-                LangServerConfigStore.installPythonLanguageServer(this.serverLocations["python"]);
+                cp.execFileSync(PYTHON_FILENAME,
```
Comment (DavidRBiderman) [inline src/langauge-servers/configStore.ts:196]: take out to its own function

**Review by Gilwe — APPROVED** [review]

File: src/langauge-servers/pythonUtils.ts [hunk 3]
```diff
@@ -59,25 +56,16 @@ const fromCommonPlaces = (): string[] => {
...
-        pythonStoreParents.forEach(pythonStoreParent => {
-            fs.readdirSync(pythonStoreParent, {withFileTypes: true}).filter(possibleStore => possibleStore.isFile()).forEach(possibleStore => {
-                if (minimatch.match([possibleStore.name], "**Python*")) {
-                    pythonLocations.push(possibleStore.name);
-                }
-            });
+        const stdout = cp.execSync("cmd /c where python", { encoding: "utf-8", stdio: ["inherit"] });
+        const trimmedOutput = _.trim(stdout);
+        const locations = _.split(trimmedOutput, /[\n\r]+/);
+        locations?.forEach(goLocation => {
```
Comment (DavidRBiderman) [inline src/langauge-servers/pythonUtils.ts:62]: I think you meant pythonLocation and not goLocation

**Pull request merged**